utils/create-jobs.py

```diff
@@ -32,14 +32,28 @@ def parse_cmdline(machines, tests, rfs_types):
                         help='job id for link creation: URLBASE/JOB_ID')
     parser.add_argument('-i', '--jobidx', dest='job_index',  action='store',
                         help='job index for link creation: URLBASE/JOB_ID/JOB_INDEX', default='1')
+    parser.add_argument('--img-name', dest='img_name',  action='store',
+                        help="img base name (such as agl-demo-platform) - require img_ext")
+    parser.add_argument('--img-ext', dest='img_ext',  action='store',
+                        help="img extension (such as ext4.xz) - require img_name")
```

Comment:
Question: maybe there's a way to add mutual requirement.
Or if not maybe we should pass it together: --image agl-demo-platform .ext4.xz 

This can be an improvement